DOCS/man/vo.rst

```diff
@@ -488,10 +488,19 @@ Available video output drivers are:
         Mode ID to use (resolution and frame rate).
         (default: 0)
 
-    ``--drm-overlay=<number>``
-        Select the DRM overlay index to use.
-        Overlay index is zero based, and related to crtc.
-        (default: 0)
+    ``--drm-osd-plane-id=<number>``
+        Select the DRM planed index to use for OSD (or OSD and video).
+        Index is zero based, and related to crtc.
+        When using this option with drm_prime renderer, it will only affect
+        the OSD contents. Otherwise will set OSD & video plane.
+        (default: first overlay plane)
+
+    ``--drm-video-plane-id=<number>``
+        Select the DRM planed index to use for video layer.
+        Index is zero based, and related to crtc.
+        This option is only valid when using the drm_prime renderer which supports
+        several layers and with ``vo=gpu`` and ``gpu-context=drm``.
```

Comment:
use commas or parentheses around "which supports several layers". 
"and with" -> "together with"

also perhaps "only valid" -> "only has an effect", since it is technically valid (i.e. mpv won't complain), it just won't do anything useful.